### PR TITLE
BACKLOG-8067 - Embed HTML in email body

### PR DESCRIPTION
**Pull request by EgorZhuk**
@tmorgner please take a look

### CODE REVIEW

**Comment by EgorZhuk** [comment]
Added support for "eml" to /api/repos

**Comment by EgorZhuk** [comment]
+ junits

**Comment by EgorZhuk** [comment]
@tmorgner this one is also ready

**Comment by tmorgner** [comment]
You have -0.22% instruction coverage change. Oh, no. :)

File: scheduler/src/org/pentaho/platform/scheduler2/email/Emailer.java [hunk 1]
```diff
@@ -166,6 +167,7 @@ public boolean setup() {
       props.put( "mail.smtp.ssl", ObjectUtils.toString( service.getEmailConfig().isUseSsl() ) );
       props.put( "mail.smtp.quitwait", ObjectUtils.toString( service.getEmailConfig().isSmtpQuitWait() ) );
       props.put( "mail.from.default", service.getEmailConfig().getDefaultFrom() );
+      props.put( "mail.debug", true );
```
Comment (tmorgner) [inline scheduler/src/org/pentaho/platform/scheduler2/email/Emailer.java]: This should either not be here or it should be a configurable option that is disabled by default.

**Comment by tmorgner** [comment]
Disable the mail.debug setting and it is ready to merge.

**Comment by EgorZhuk** [comment]
@tmorgner I doubt that removing mail.debug affects mondian :)

**Comment by tmorgner** [comment]
Yeah, reporting-plugin isn't the only project suffering from flaky unit tests.

**Pull request merged**